tools/cli/help.txt

```diff
@@ -1,19 +1,25 @@
 >>>
 (the first line of each entry is the summary for autogenerated command lists)
-Usage: meteor [--release <release>] [--help] <command> [args]
-       meteor help <command>
+Usage: meteor [--release <release>] [--help] <built-in command> [args]
+       meteor help <built-in command>
+       meteor <external command> [args]       
        meteor [--version] [--arch]
+       
 
 With no arguments, 'meteor' runs the project in the current
 directory in local development mode. You can run it from the root
 directory of the project or from any subdirectory.
 
 Use 'meteor create <path>' to create a new Meteor project.
 
-Commands:
+Built-in Commands:
 {{commands}}
 
-See 'meteor help <command>' for details on a command.
+See 'meteor help <built-in command>' for details on a command.
+
+You can use 'meteor <external command> [args]' to run a binary command
+installed previously with 'meteor npm install -g <package with command>',
+which will store the package under the global meteor installation folder.
```

Comment:
Thanks for the suggestions. Point after point:
- True, I used -g because it is shorter, but makes sense to use --global to make it clear since we are in the context of helping the user. 
- Yep, the example thing was on my mind, but was not sure if too much text was something good on the help section (and maybe was more suitable for the guide). Anyway `yarn` and `jsdoc` were my two candidates for an example, so yarn is fine by me.
- I know, we can use commands all alone to avoid missunderstandings if that helps then.

Besides all of this, I still think that a more in-depth explanation about node, npm and included tools (python, mongo, etc...) could be added to the guide too, since it is important to mention for example that with every new meteor release, we should install the npm commands again, and that if we are looking for the place were meteor stores them, is it in the dev_bundle folder.